### PR TITLE
style contact us #40

### LINKED ISSUE #40: style contact us
#9
- [ ] style contact us

### PR DESCRIPTION
**Pull request by walaamedhat**
- fixes #40 

- related to #40  

- make contact us page html,css

### CODE REVIEW

File: public/aboutus.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    <header>
+      <form id="headerForm" class="headerForm" action="" method="post">
+        <input type="submit" name="" value="Login">
+        <input type="submit" name="" value="Sign Up">
+      </form>
+    </header>
+      <div class="burgerAndLogo" id="burgerAndLogo">
+        <div></div>
+    <div><img src="./imgs/logo.png" alt="logo" class="logo"></div>
+    <div class="burgerButton">
```
Comment (MahmoudMH) [inline public/aboutus.html:23]: change it to icon .

**Review by MahmoudMH — CHANGES_REQUESTED** [review]

File: public/aboutus.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      <h1>About Us</h1>
+      <center><div class="parag"><p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem.</p>
+    </div></center>
+    <center>
+    <div class="row">
+  <div class="column" >
+    <img src="./imgs/walaa.png" alt="walaa">
+  </div>
+  <div class="column" >
+    <img src="./imgs/mahmoud.jpeg" alt="walaa">
```
Comment (Salam-Dalloul) [inline public/aboutus.html:40]: the alt should be mahmoud! :smile_cat:

File: public/css/aboutus.css [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+input {
+  background-color: #742223;
+  border: 0;
+  color: white;
+  font-family: Roboto;
+  font-style: Regular;
+  font-size: 15px;
+}
+
+input:hover {
```
Comment (Salam-Dalloul) [inline public/css/aboutus.css:31]: it's better to use classes :D

**Review by Salam-Dalloul — CHANGES_REQUESTED** [review]

File: public/aboutus.html (hunk 1 above)
Comment (walaamedhat) [reply public/aboutus.html:23]: it modified in #61

File: public/aboutus.html (hunk 2 above)
Comment (walaamedhat) [reply public/aboutus.html:40]: it modified in #61

File: public/css/aboutus.css (hunk 3 above)
Comment (walaamedhat) [reply public/css/aboutus.css:31]: this issue for contact us not for about us

**Review by muhmushtaha — APPROVED** [review]

**Pull request merged**